### PR TITLE
Make Enzyme v0.0.130 cache appear

### PR DESCRIPTION
**Pull request by tzunghanjuang**
**Context:**
As title. I think enzyme version update (to v0.0.149) makes stable version break. Hopefully this can fix.

### CODE REVIEW

File: .github/workflows/check-catalyst-enzyme-v0.0.130.yaml
Comment (dime10) [inline .github/workflows/check-catalyst-enzyme-v0.0.130.yaml:1]: Can you reduce this to generating the missing cache entries and nothing else?

Comment (tzunghanjuang) [reply .github/workflows/check-catalyst-enzyme-v0.0.130.yaml:1]: Yes I have reduced it to LLVM and Enzyme only.

File: .github/workflows/build-enzyme-v0.0.130.yaml [hunk 1]
```diff
@@ -0,0 +1,161 @@
+name: Build Enzyme v0.0.130
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
```
Comment (dime10) [inline .github/workflows/build-enzyme-v0.0.130.yaml:9]: Don't need this to run on PRs right? 
```suggestion
```

**Review by dime10 — APPROVED** [review]
Thanks for fixing the matrix!

File: .github/workflows/build-enzyme-v0.0.130.yaml (hunk 1 above)
Comment (tzunghanjuang) [reply .github/workflows/build-enzyme-v0.0.130.yaml]: Yes we don't need this.

**Pull request merged**